recipes/llm-voice-assistant/ios/PicoLLMVoiceAssistantDemo/PicoLLMVoiceAssistantDemoApp.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  Copyright 2024 Picovoice Inc.
+//  You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+//  file accompanying this source.
+//  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+//  specific language governing permissions and limitations under the License.
+//
+
+import SwiftUI
+
+@main
+struct PicoLLMVoiceAssistantDemoApp: App {
```

Comment:
I think these cookbook demos aren't supposed to be as branded and picoLLM focussed (see the readme in the llm-voice-assistant dir 'LLM-Powered Voice Assistant'). I think we should just call this demo LLM Voice Assistant, like the demo directory is named.